kalite/distributed/static/js/distributed/rating/views.js

```diff
@@ -108,6 +108,12 @@ module.exports = BaseView.extend({
             },
             patch: true
         });
+    },
+
+    close: function() {
+        // Defined so that when BaseView.remove is called, we just clear the html instead

```

Comment:
This view's `el` property is assigned to an element in the template of the parent view. When `remove` is called this view's `el` is removed from the DOM. What I have chosen to do instead is clear the html of the `el` so that it remains attached to the DOM. Then new instantiations of RatingViews can be attached to the same element